src/polyrith.lean

```diff
@@ -15,17 +33,14 @@ inductive poly
 |var: ℕ → poly
 |add: poly → poly → poly
 |mul: poly → poly → poly
+|pow : poly → ℕ → poly
```

Comment:
I noticed that the certificates `polyrith` produced were expanding powers into multiplication, which doesn't format very well. Better to keep the exponents, and reflect this in the `poly` datatype!